compiler/rustc_codegen_cranelift/example/std_example.rs

```diff
@@ -176,7 +176,7 @@ fn main() {
         None
     }
 
-    static STATIC_WITH_MAYBE_NESTED_BOX: &Option<Box<str>> = &no_str();
+    static STATIC_WITH_MAYBE_NESTED_BOX: Option<&Box<str>> = &no_str();
```

Comment:
Please take a look at this one - I am not sure if this is correct or not.

Reply:
This is a test. It shouldn't be touched unless necessary.

Reply:
thx, reverted